### ISSUE TITLE
Error 404 updating

### ISSUE THREAD

**Issue by cibernicola** [rabuchaim/geoip2fast#12]
Using this snippet:

```
from geoip2fast import GeoIP2Fast
G = GeoIP2Fast()
G.update_file('geoip2fast.dat.gz',verbose=True)
G.reload_data(verbose=True)

```
This is the result:

```
- Opening URL https://github.com/rabuchaim/geoip2fast/releases/download/LATEST/geoip2fast.dat.gz
- Error downloading file: HTTP Error 404: Not Found - https://github.com/rabuchaim/geoip2fast/releases/download/LATEST/geoip2fast.dat.gz
- Error downloading file: HTTP Error 404: Not Found - https://github.com/rabuchaim/geoip2fast/releases/download/LATEST/geoip2fast.dat.gz
GeoIP2Fast v1.2.2 is ready! geoip2fast.dat.gz loaded with 973.060 networks in 0.24206 seconds and using 9.78 MiB.
```

https://github.com/rabuchaim/geoip2fast/releases/download/LATEST/geoip2fast.dat.gz  <-404

**Comment by ddxv** [comment]
Yeah, it looks like there are no files in the release LATEST tag?

https://github.com/rabuchaim/geoip2fast/releases/tag/LATEST

But I do see they've recently been updated here:

https://github.com/rabuchaim/geoip2fast/tree/main/geoip2fast

So maybe just weren't tagged @rabuchaim ?

**Comment by rabuchaim** [comment]
Sorry guys!! My GitHub token expired and therefore the automatic release update did not complete. 

![image](https://github.com/user-attachments/assets/cd26a177-886f-4773-ad01-c41d50ae740e)

I have already renewed the token and now it has no expiration date so this will not happen again.

The new releases were generated successfully!

![image](https://github.com/user-attachments/assets/96b1a127-6d98-4cd8-aaa2-544e9e924fee)

[..]s Ricardo Abuchaim

**Comment by ddxv** [comment]
Ah, yes, this will happen to me soon too. Thanks for the reminder
